src/Kit/System/Watchdog/WatchedEventThread.cpp

```diff
@@ -0,0 +1,91 @@
+/*------------------------------------------------------------------------------
+ * Copyright Integer Fox Authors
+ *
+ * Distributed under the BSD 3 Clause License. See the license agreement at:
+ * https://github.com/Integerfox/kit.core/blob/main/LICENSE
+ *
+ * Redistributions of the source code must retain the above copyright notice.
+ *----------------------------------------------------------------------------*/
+/** @file */
+
+#include "Kit/System/Watchdog/WatchedEventThread.h"
+#include "Kit/System/EventLoop.h"
+#include "Kit/System/Assert.h"
+
+using namespace Kit::System;
+
+/////////////////////////
+WatchedEventThread::WatchedEventThread( uint32_t wdogTimeoutMs, uint32_t healthCheckIntervalMs, bool isSupervisor ) noexcept
+    : WatchedThread( wdogTimeoutMs )
+    , m_timer( *this, &WatchedEventThread::healthTimerExpired )
+    , m_healthCheckIntervalMs( healthCheckIntervalMs )
+    , m_isSupervisor( isSupervisor )
+    , m_isActive( false )
+{
+    KIT_SYSTEM_ASSERT( wdogTimeoutMs > healthCheckIntervalMs );
+}
+
+void WatchedEventThread::startWatcher( Kit::System::EventLoop& eventLoop ) noexcept
+{
+    if ( m_isActive )
+    {
+        return;  // Already started
+    }
+    // Set the timing source for the notify timer
+    m_timer.setTimingSource( eventLoop );
+
+    // Register this thread with the supervisor
+    Supervisor::beginWatching( *this );
+    // Start the health check timer
+    m_timer.start( m_healthCheckIntervalMs );
+    m_isActive = true;
```

Comment:
This is a **white space** only change.  Yes - I know I am being OCD - but I really believe visually pretty code has value, i.e. easier to maintained.  I have also updated the Coding Standard to more explicit on this topic.
```suggestion
    if ( m_isActive )
    {
        return;  // Already started
    }
    
    // Set the timing source for the notify timer
    m_timer.setTimingSource( eventLoop );

    // Register this thread with the supervisor
    Supervisor::beginWatching( *this );
    
    // Start the health check timer
    m_timer.start( m_healthCheckIntervalMs );
    m_isActive = true;
```